docs/self-managed/components/management-identity/configuration/identity-configuration-overview.md

```diff
@@ -7,6 +7,12 @@ description: "Learn more about how Management Identity plays together with Keycl
 
 Configure Management Identity for your Camunda 8 Self-Managed deployment.
 
+:::info Deploying with Helm?
```

Comment:
Should we add the note also the subpages "Connect to an existing Keycloak instance" and "Configure an external IdP using Keycloak"? We could even link directly to the equivalent Helm guides, for example to https://docs.camunda.io/docs/self-managed/deployment/helm/configure/authentication-and-authorization/external-keycloak/ for the existing Keycloak instance page.